<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1157</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spectrorgam Viewer</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>File name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>File path</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="6">
       <widget class="QGroupBox" name="gbViewMode">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="title">
         <string>View mode</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QRadioButton" name="rbTimeLimitsMode">
             <property name="text">
              <string>By time limits</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbTimeStepMode">
             <property name="text">
              <string>By time step</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Start time analysis</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateTimeEdit" name="dtStartTime">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="displayFormat">
              <string>dd.MM.yyyy HH:mm:ss</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Stop time analysis</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateTimeEdit" name="dtStopTime">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="displayFormat">
              <string>dd.MM.yyyy HH:mm:ss</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>Time step size (minutes)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sbTimeStepSize">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maximum">
              <number>120</number>
             </property>
             <property name="value">
              <number>60</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>Step number</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sbTimeStepIndex">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bShow">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Show</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="bLoadData">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="6">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Origin signal parameters</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Component</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbComponents">
             <item>
              <property name="text">
               <string>Z</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>X</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Y</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Resample frequency</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sbResampleFrequency">
             <property name="maximum">
              <number>10000</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="leFilePath">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="pbOpenFile">
        <property name="text">
         <string>ðŸ“ƒ</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbFileName"/>
      </item>
      <item row="2" column="0" colspan="6">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Spectrogram visualization parameters</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Minimal frequency</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="dsSpectrogramMinFrequency">
             <property name="maximum">
              <double>10000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Maximal frequency</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="dsSpectrogramMaxFrequency">
             <property name="maximum">
              <double>10000.000000000000000</double>
             </property>
             <property name="value">
              <double>20.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Signal</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="PlotWidget" name="gwGraphOriginalSignal"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Spectrogram</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="GraphicsLayoutWidget" name="gwGraphOriginalSpectrogram"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1157</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph.h</header>
  </customwidget>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
